### PR TITLE
Added changelog for 0.13

### PR DESCRIPTION
**Pull request by aquamatthias**
@philipnrmn please also have a look as our English representative :)

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -29,6 +42,24 @@ Since this topic has so distinct requirements for different organizations, it is
 This version now implements all the necessary hooks needed to secure most external interfaces to your specific needs.
 If you are interested in a very simple implementation, you can look into the [Example Auth Plugin](https://github.com/mesosphere/marathon-example-plugins/tree/master/auth)
 
+#### Persistent Store Cache 
+All entities in the persistent store (zk) is loaded into a cache during leader election.
```
Comment (gkleiman) [inline changelog.md]: s/(zk) is/(ZK) are/

**Comment by gkleiman** [comment]
LGTM, assigning to our official English representative ;-)

File: changelog.md [hunk 2]
```diff
@@ -9,6 +9,19 @@ do no longer contain redundant information about the app id. Additionally, the t
...
 
+#### Zookeeper Compression
+ZK nodes larger than a certain threshold will now be compressed. This allows Marathon to handle
+more apps and groups, but breaks backwards compatibility, because older versions of Marathon are
+not able to parse compressed nodes. You can define the threshold with `--zk_compression_threshold` 
+which defaults to 64KB. 
+To disable this feature, start Marathon with the `--disable-zk-compression` flag.
+
+#### Use logback as logging backend
+We moved from Log4j to [Logback](http://logback.qos.ch) backend.
```
Comment (philipnrmn) [inline changelog.md]: log4j is styled lowercase

File: changelog.md [hunk 3]
```diff
@@ -29,6 +42,24 @@ Since this topic has so distinct requirements for different organizations, it is
 This version now implements all the necessary hooks needed to secure most external interfaces to your specific needs.
 If you are interested in a very simple implementation, you can look into the [Example Auth Plugin](https://github.com/mesosphere/marathon-example-plugins/tree/master/auth)
 
+#### Persistent Store Cache 
+All entities in the persistent store (ZK) are loaded into a cache during leader election.
+Subsequent reads are delivered from that cache. Updates to entities also update the cache.
+This cache should improve read access time significantly.
+Please be aware that the time until the leader takes over leadership will be increased with a lot of apps and tasks.
```
Comment (philipnrmn) [inline changelog.md]: I don't understand what exactly this time period is, can we clarify it a bit? Is it the time between the old leader becoming unavailable and the new leader becoming available?

File: changelog.md [hunk 4]
```diff
@@ -29,6 +42,24 @@ Since this topic has so distinct requirements for different organizations, it is
...
+All entities in the persistent store (ZK) are loaded into a cache during leader election.
+Subsequent reads are delivered from that cache. Updates to entities also update the cache.
+This cache should improve read access time significantly.
+Please be aware that the time until the leader takes over leadership will be increased with a lot of apps and tasks. 
+You can disable the cache with `--disable_store_cache`.
+
+#### Greatly improved API Reference
+With this release we integrated [RAML](http://raml.org) based API documentation.
+The `/help` endpoint uses the [RAML Console](https://github.com/mulesoft/api-console) to show the API Reference.
+Also the GitHub pages uses that specification.
```
Comment (philipnrmn) [inline changelog.md]: The Github pages documentation now also uses that specification.

File: changelog.md [hunk 5]
```diff
@@ -29,6 +42,24 @@ Since this topic has so distinct requirements for different organizations, it is
...
+We had a lot of feedback for documentation improvements - so please give us your thoughts on that. 
+
+#### Graphite and DataDog reporter
+We collect a lot of metrics in Marathon.
+You can collect those metrics via the `/metrics` endpoint.
+With those reporters you can transfer the data into either Graphite or DataDogs and see the values over time.
```
Comment (philipnrmn) [inline changelog.md]: Should DataDog be singular here?

File: changelog.md [hunk 6]
```diff
@@ -61,26 +97,54 @@ We now publish a separate marathon-plugin-interface.jar with every Marathon rele
 This artifact holds all the inerfaces needed to develop your own Marathon plugin.
 
 #### Consolidate logging to use slf4j
-We moved completely to slf4j as Logging API. The logging backend still uses log4j.
+We moved completely to slf4j as Logging API. 
+
+#### Several performance improvements
+* A separate thread pool is used for health check operations
+* Efficiently create groups/apps
```
Comment (philipnrmn) [inline changelog.md]: Group and app creation is more efficient.
(just a stylistic tweak to make it match the context)

File: changelog.md (hunk 3 above)
Comment (gkleiman) [inline changelog.md]: After the current leader loses his leadership, there's a leaderless lapse of time until the next leader is elected. During that time Marathon is leaderless and no API calls are accepted.

The new behaviour makes an elected candidate load the cache before becoming leader, which means that the duration of the leaderless period can be longer if there are many entities to load.

File: changelog.md [hunk 7]
```diff
@@ -61,26 +97,54 @@ We now publish a separate marathon-plugin-interface.jar with every Marathon rele
...
+
+#### Several performance improvements
+* A separate thread pool is used for health check operations
+* Efficiently create groups/apps 
 
 #### Introduce configurable zk node compression
 We introduced zk compression which improves performance significantly. Compression can be turned on/off via cmd line flags and is enabled by default.
 
+#### Tasks are now stored via the standard EntityRepository
+The storage of tasks was always handled specifically in Marathon.
```
Comment (philipnrmn) [inline changelog.md]: specifically how?

File: changelog.md [hunk 8]
```diff
@@ -61,26 +97,54 @@ We now publish a separate marathon-plugin-interface.jar with every Marathon rele
...
+With that change in place we handle all entities via the same interface.
```
Comment (philipnrmn) [inline changelog.md]: s/that change/this change/

File: changelog.md (hunk 3 above)
Comment (philipnrmn) [inline changelog.md]: Great, let's use the second sentence of that explanation verbatim :+1:

Comment (aquamatthias) [inline changelog.md]: Too confusing - will remove that sentence.

File: changelog.md (hunk 5 above)
Comment (aquamatthias) [inline changelog.md]: Correct.

File: changelog.md (hunk 6 above)
Comment (aquamatthias) [inline changelog.md]: Better.

File: changelog.md (hunk 7 above)
Comment (aquamatthias) [inline changelog.md]: separately?

Comment (philipnrmn) [inline changelog.md]: Ah, I see. In that case I suggest:
The storage of tasks was handled separately in previous versions of Marathon.

**Comment by aquamatthias** [comment]
@philipnrmn Thanks for the suggestions! Hope I addressed them all.

**Comment by philipnrmn** [comment]
LGTM :+1:

**Pull request merged**